src/test/test_timer.cpp

```diff
@@ -24,7 +24,7 @@ SISL_OPTION_GROUP(test_timer,
                    ::cxxopts::value< uint32_t >()->default_value("4"), "number"),
                   (user_threads, "", "user_threads", "user_threads", ::cxxopts::value< uint32_t >()->default_value("2"),
                    "number"),
-                  (num_timers, "", "num_timers", "num_timers", ::cxxopts::value< uint64_t >()->default_value("1000"),
+                  (num_timers, "", "num_timers", "num_timers", ::cxxopts::value< uint64_t >()->default_value("300"),
```

Comment:
Please do not reduce the num_timers. It is 1000 to test several cases. The problem of not enough file descriptors is your environment. Please increase ulimit of file descriptors to 65536.